src/rust-crypto/rust-crypto.ts

```diff
@@ -137,6 +144,9 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, RustCryptoEv
             onCrossSigningKeysImport,
         );
     }
+    public async deleteKeyBackupVersion(version: string): Promise<void> {
+        await this.backupManager.deleteKeyBackupVersion(version);
+    }
```

Comment:
this seems to be out of place?

Reply:
yes but why is it next to the constructor, rather than alongside the other backup functions?